security/sshdo/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                sshdo
+version             1.1
+revision            0
+
+categories          security
+license             GPL-2+
+maintainers         {raf.org:raf @macportsraf} openmaintainer
+description         controls which commands may be executed via incoming ssh
+long_description    {*}${description}. \
+                    \n\nsshdo provides an easily configurable way of controlling which \
+                    commands may be executed via incoming ssh connections by specific \
+                    users and specific keys. Training mode allows a key to be used for \
+                    any command, while logging them. Then sshdo can learn from the logs \
+                    to know what commands to allow. It can also unlearn commands that \
+                    are no longer in use. This mitigates against private ssh key compromise \
+                    with very little effort.
+
+github.setup        raforg ${name} ${version} v
```

Comment:
move this to the top of the Portfile and incorporate the "name" and "version" fields in the "github.setup" line

Reply:
Thanks, but I'd really rather not. Unless there's a compelling reason.

Reply:
OK. I've changed it to download from the homepage instead. Is that OK?

Reply:
Aesthetics. Never mind. I thought it was only conventions relating to the use of the github port group that concerned you, and that by not using that port group, we could both get what we want. I've changed it so that the github.setup is near the top.